README.md

```diff
@@ -468,6 +469,158 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+If you need the full power of an SDK component, you can specificy a custom Direct Answers card.
```

Comment:
I'm not sure I follow this opening sentence. We want to highlight that you can fully customize/configure the look of your DA with a custom DA card.

Reply:
got it updated